tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorDataType.spec.ts

```diff
@@ -153,14 +150,17 @@ test.describe('BlockListEditorDataType', () => {
   });  
 
   test('can edit a block list editor', async ({page, umbracoApi, umbracoUi}) => {
+    // We need to increase the timeout because the test is taking too long to end
+    await page.setDefaultTimeout(60000);
```

Comment:
shouldn't this be `test.setTimeout(60000);`

Reply:
Or instead, use `testInfo.slow` is probably better: https://playwright.dev/docs/api/class-testinfo#test-info-slow-1